pandas/core/groupby/groupby.py

```diff
@@ -548,6 +549,29 @@ def __repr__(self) -> str:
         # TODO: Better repr for GroupBy object
         return object.__repr__(self)
 
+    def _repr_html_(self) -> str:
```

Comment:
I'd like to reorg this to use a GroupbyFormatter located in pandas/io/formats/groupby.py (it can do pretty much this but just locate the code there) as this is where we keep all of the formatting code.

could *also* add a `.to_string()` method but not sure that's actually worth it (maybe open an issue for that).

Reply:
Thank for the review! Do you mean pandas/io/formats/html.py? Should I add a new function and then just call that function from the above location?

Reply:
no, i mean pandas/io/formats/format.py (ok to just shove in there is fine, we should split that file up but that's for later).